nservicebus/scheduling-with-nservicebus.md

```diff
@@ -25,25 +25,20 @@ The difference between these examples is that in the latter a name is given for
 
 <!-- import ScheduleTask -->
 
-## Accuracy 
-
-The scheduling infrastructure leverages the reliable messaging approach at the core of NServiceBus. This allows scheduling to include features such as the error queue and retries. This however does have some side effects on the timeliness of scheduled tasks. When a task is scheduled to be run at a given time it is actual not "executed at that time", it is instead "queued at that time". In most cases this distinction will have no noticeable effect on the behavior of the the scheduling API. However in high load systems the fact that a scheduled task is added to the back of the queue can result in a noticeable delay between the "time the task has been request to be run" and the "time the task is actually executed".
-
-## Implementation
-
-The scheduler holds a all list of tasks scheduled. In version 3 and 4 tasks are scoped per `AppDomain`. In version 5 they are scoped per Bus instance.
+## Current Limitations
 
-When the task is created it is given an unique identifier. The identifier for the task is sent in a message to the Timeout Manager. When it times out and the Timeout Manager returns the message containing the identifier to the endpoint with the scheduled task, the endpoint uses that identifier to fetch and invoke the task from its internal list of tasks.
-
-## Assumptions
-
-- Scheduled tasks are created at endpoint startup using for example a class the implements the `IWantToRunWhenBusStartsAndStops` interface;
-- If the process restarts, all scheduled tasks are recreated and given new identifiers. Tasks scheduled before the restart will not be found and a message is written to the log. This is expected behavior;
-- A scheduled task will run forever, as long as the endpoint is running, there is no way to cancel it except skipping its execution each time is invoked;
+- Since the scheduler is non-durable, if the process restarts, all scheduled tasks (that are created during the endpoint's startup) are recreated and given new identifiers. Tasks scheduled before the restart that arrive at the endpoint queue will not be found and a message is written to the log as information.  
+- Already created schedule tasks cannot be canceled or changed.
+- Does not support a task to be started at a specified specific time.
 - Each scheduled task executes on a new thread using the `Task.Factory.StartNew(Action)` method, which means that there will be no transaction scope by default and it is up to you to create one if needed;

```

Comment:
@mauroservienti - will change it per your suggestion. I originally kept the statement as is (from the original doco).